src/app/components/BibPage/BibContainer.jsx

```diff
@@ -0,0 +1,42 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import { useBib } from '../../context/Bib.Provider';
+import LibraryItem from '../../utils/item';
+import { getElectronics, pluckAeonLinks } from '../../utils/utils';
+import LegacyCatalogLink from '../LegacyCatalog/LegacyCatalogLink';
+import BibHeading from './BibHeading';
+import BottomBibDetails from './BottomBibDetails';
+import BibHoldings from './components/BibHoldings';
+import BibItems from './components/BibItems';
+import TopBibDetails from './TopBibDetails';
+
+const BibContainer = ({ location, keywords, selection }) => {
+  const { bib, bibId } = useBib();
+
+  const items = (bib.checkInItems || []).concat(LibraryItem.getItems(bib));
+  const resources = getElectronics(items);
+
+  return (
+    <>
+      <BibHeading searched={selection} />
```

Comment:
This is the original format (you can find this in `BibPage.jsx`) and converted to what you see now in `BibContainer`. Think of this file as defining the sections of items to be rendered to the DOM and not it's layout (where the section element is declared). Eitherway, I'd argue in favor of this approach rather then converting back to what you have here. I feel it's more legible, precise and declarative. It also allows each item/section/node/component to define it's own layout rather than the the Container  where it's pupose is to define what it wants in the DOM.

Reply:
OK, this is also a preference thing.  If you feel strongly, I don't mind.  However, if sections should have a consistent margin, I think it's best to extract `{{ marginTop: '20px' }}` out into something external so that there isn't a need to update four files when someone decides to change the margin.

Reply:
I agree. I believe the best option here is to make a Jira Ticket to handle this at a future date.